android/lib/src/main/java/com/swmansion/gesturehandler/PanGestureHandler.java

```diff
@@ -122,20 +139,20 @@ public PanGestureHandler setMinVelocityY(float minVelocityY) {
 
   private boolean shouldActivate() {
     float dx = mLastX - mStartX + mOffsetX;
-    if (mMinDeltaX != MIN_VALUE_IGNORE && Math.abs(dx) >= mMinDeltaX) {
+    if (mMinOffsetRangeStartX != MIN_VALUE_IGNORE && dx <=  mMinOffsetRangeStartX) {
```

Comment:
I don't think this is quite correct, if the minOffsetRange is (-50, -20) it means I only want to activate when pulled up by 20px but because of that it will activate immediately because initial offset of 0 is greater than -50